README.md

```diff
@@ -8,3 +8,48 @@ This project is not setup for contributing code. It is intended only for trackin
+|  | [AAD Pod Identity](https://github.com/Azure/aad-pod-identity) <br> ([releases](https://github.com/Azure/aad-pod-identity/releases)) | Enables K8s applications to access cloud resources securely with Azure Active Directory | Azure: incubation <br> API: v1 | [GitHub issues](https://github.com/Azure/aad-pod-identity/issues) <br> [GitHub Project](https://github.com/Azure/aad-pod-identity/projects/3) | [Use with AKS](https://docs.microsoft.com/en-us/azure/aks/developer-best-practices-pod-security#use-pod-managed-identities) <br> [Use with AKS Engine](https://github.com/Azure/aks-engine/tree/master/examples/addons/aad-pod-identity) |
+|  | [OPA Gatekeeper](https://github.com/open-policy-agent/gatekeeper) <br> ([releases](https://github.com/open-policy-agent/gatekeeper/releases)) | K8s native Open Policy Agent policy enforcement | CNCF: incubating <br> API: alpha | [#kubernetes-policy](https://openpolicyagent.slack.com/archives/CDTN970AX) <br> [GitHub issues](https://github.com/open-policy-agent/gatekeeper/issues) | [Azure Policy for AKS](https://docs.microsoft.com/en-us/azure/governance/policy/concepts/rego-for-aks) <br> [Azure Policy for AKS Engine](https://docs.microsoft.com/en-us/azure/governance/policy/concepts/aks-engine) |
+|  | [Keyvault Flexvol](https://github.com/Azure/kubernetes-keyvault-flexvol) <br> ([releases](https://github.com/Azure/kubernetes-keyvault-flexvol/releases)) | Mount Azure Keyvault secrets into pods | Azure: incubation <br> API: v1 | [GitHub issues](https://github.com/Azure/kubernetes-keyvault-flexvol/issues) | [Use with AKS](https://docs.microsoft.com/en-us/azure/aks/developer-best-practices-pod-security#use-azure-key-vault-with-flexvol) |
+|  | [Secrets Store CSI Driver](http://sigs.k8s.io/secrets-store-csi-driver) <br> ([releases](https://github.com/kubernetes-sigs/secrets-store-csi-driver/releases)) <br> [Builds](https://testgrid.k8s.io/sig-auth-secrets-store-csi-driver#secrets-store-csi-driver-e2e-vault-postsubmit) | Integrates secrets stores with Kubernetes via a [Container Storage Interface (CSI)](https://kubernetes-csi.github.io/docs/) volume | CNCF: ?? <br> API: v1 | [GitHub issues](https://github.com/kubernetes-sigs/secrets-store-csi-driver/issues) |  |
+|  | [Azure KeyVault Provider for Secrets Store CSI Driver](https://github.com/Azure/secrets-store-csi-driver-provider-azure) <br> ([releases](https://github.com/Azure/secrets-store-csi-driver-provider-azure/releases)) | Enables mounting AKV secrets as volumes in K8s pods | Azure: sandbox <br> API: v1 | [GitHub issues](https://github.com/Azure/secrets-store-csi-driver-provider-azure/issues) |  |
+| Cloud Native Service Mesh |  |  |  |  |
+|  | [Service Mesh Interface (SMI) Spec](https://smi-spec.io/) | A standard interface for service meshes on Kubernetes | Azure: sandbox <br> API: N/A | [#general](https://smi-spec.slack.com/archives/CJJF5M5NK) <br> [GitHub issues](https://github.com/deislabs/smi-spec/issues) |  |
```

Comment:
cc @michelleN to help provide API versions for SMI resources

Reply:
Should be CNCF Sandbox I think: https://github.com/cncf/toc/pull/336
SMI Spec API versions:
- Traffic Split v1alpha3
- Traffic Target v1alpha1
- Traffic Spec v1alpha2
- Traffic Metrics v1alpha1

Reply:
The merge is still pending :) I'll include it. Thanks for the API versions